tests/parser/AquiferCTTests.cpp

```diff
@@ -68,11 +104,21 @@ inline std::vector<AquiferCT::AQUCT_data> init_aquiferct(){
 }
 
 BOOST_AUTO_TEST_CASE(AquiferCTTest){
-    std::vector< AquiferCT::AQUCT_data > aquiferct = init_aquiferct();
+    auto deck = createAquiferCTDeck();
+    std::vector< AquiferCT::AQUCT_data > aquiferct = init_aquiferct(deck);
+    for (const auto& it : aquiferct){
+        BOOST_CHECK_EQUAL(it.aquiferID , 1);
+        BOOST_CHECK_EQUAL(it.phi_aq , 0.3);
+        BOOST_CHECK_EQUAL(it.inftableID , 2);
+        BOOST_CHECK_CLOSE(*(it.p0), 1.5e5, 1e-6);
+    }
+
+    auto deck_default_p0 = createAquiferCTDeckDefaultP0();
+    aquiferct = init_aquiferct(deck_default_p0);
     for (const auto& it : aquiferct){
         BOOST_CHECK_EQUAL(it.aquiferID , 1);
         BOOST_CHECK_EQUAL(it.phi_aq , 0.3);
         BOOST_CHECK_EQUAL(it.inftableID , 2);
-        
-    }    
+        BOOST_CHECK(it.p0 == false);
+    }
```

Comment:
@joakim-hove Any idea why the jenkins build failed for the it.p0 == false check? As I understood, it.p0 should be a nullptr, hence would evaluate to a false?

Reply:
What is the local failure message? or it passed in your local test?

Reply:
It evaluates as not `true`, but that is not the same as ` == false`. Just use: `if (!p0)` or `if (p0 == nullptr) `